keyboards/era/linx3/n8x/info.json

```diff
@@ -0,0 +1,135 @@
+{
+    "manufacturer": "eerraa",
+    "keyboard_name": "N8X",
+    "maintainer": "eerraa",
+    "backlight": {
+        "breathing": true,
+        "breathing_period": 5,
+        "levels": 10,
+        "pin": "GP3"
+    },
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": true,
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "GP11",
+        "scroll_lock": "GP8"
+    },
+    "matrix_pins": {
+        "cols": ["GP29", "GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP22", "GP20", "GP19", "GP18", "GP17", "GP16", "GP21", "GP1", "GP0", "GP2"],
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP10", "GP9"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0007",
+        "vid": "0x4552"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
Does this PCB have more than one configurable physical layout?

e.g.
- 2u or split Backspace
- 2.75u or split Right Shift
- ANSI and ISO specific keys
- Tsangan/WK bottom row

Reply:
> Does this PCB have more than one configurable physical layout?
> 
> e.g.
> 
> * F12 or F13 top row
> * 2u or split Backspace
> * 2.75u or split Right Shift
> * ANSI and ISO specific keys
> * Tsangan/WK bottom row

This keyboard has four physical layouts corresponding to the items below:
* 2u or split Backspace
* 2.75u or split Right Shift
* ANSI and ISO specific keys
* Tsangan/WK bottom row

However, all of these layouts can be operated as one layout, so I did not add them separately.
Should I add this to each layout?

Reply:
> However, all of its physical layouts can be operated as one layout, so I did not add them separately.
Should I add this to each layout?

In the Keyboard sub-section of the PR Checklist are guidelines regarding more than one physical layout, though to help expedite I'll add code suggestion(s) 😉

Reply:
The matrix for 2u Backspace is (0,14).
and left split backspace is (0,13) right split backspace is (0,14) :)